tst/EnergyPlus/unit/Fixtures/EnergyPlusFixture.cc

```diff
@@ -290,6 +291,20 @@ bool EnergyPlusFixture::has_dfs_output(bool reset_stream)
     return has_output;
 }
 
+bool EnergyPlusFixture::match_err_stream(std::string const &expected_match, bool use_regex, bool reset_stream)
+{
+    auto const stream_str = this->err_stream->str();
+    bool match_found;
+    if (use_regex) {
+        match_found = std::regex_match(stream_str,std::regex(expected_match));
+    } else {
+        match_found = stream_str.find(expected_match) != std::string::npos;
+    }
+    EXPECT_TRUE(match_found);
```

Comment:
This line may need to be removed depending on what we decide to do with `ENABLE_GTEST_DEBUG_MODE`.